media/mojo/mojom/BUILD.gn

```diff
@@ -123,6 +127,10 @@ mojom("mojom") {
     enabled_features += [ "enable_cast_renderer" ]
   }
 
+  if (is_cobalt && use_starboard_media) {
```

Comment:
Pending GN review, I think this can be just `if (use_starboard_media) {`, and above doesnt' need `if (is_cobalt)`. Same throughout.

Reply:
This is done correctly as is, see go/cobalt-naming#gn for an example. I can expand more on that section if needed

Reply:
Hmm, that's interesting , looking at go/cobalt-naming#values-for-common-variables `is_cobalt` seems redundant in this case. 

I do see we have a naming system of (is_cobalt && use_foo) - I would argue for removing the redundant flag when needed. But that's a separate discussion - shouldn't block this PR

Reply:
`is_cobalt` is not redundant, as if `is_cobalt = false`, `use_starboard_media` is undefined.